docs/api_static/script.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+function addLink(element) {
+  const a = document.createElement("a");
```

Comment:
this would not work on all old browsers, maybe a time to dogfood bucklescript it self ; 0